aiida/manage/manager.py

```diff
@@ -445,12 +445,18 @@ def check_rabbitmq_version(communicator: 'RmqThreadCommunicator'):
     from packaging.version import parse
 
     from aiida.cmdline.utils import echo
+    from aiida.manage.configuration import get_config_option
+
+    show_warning = get_config_option('warnings.rabbitmq_version')
```

Comment:
```suggestion
    show_warning = self.get_option('warnings.rabbitmq_version')
```

Reply:
(FYI this is the code of `get_config_option`: `return get_manager().get_option(option_name)`)

Reply:
Done